pandas/tests/arrays/test_timedeltas.py

```diff
@@ -9,6 +9,18 @@
 
 
 class TestTimedeltaArrayConstructor(object):
+    def test_only_1dim_accepted(self):
+        # GH#25282
+        arr = np.array([0, 1, 2, 3], dtype='m8[h]').astype('m8[ns]')
+
+        with pytest.raises(ValueError, match="Only 1-dimensional"):
+            # 2-dim
+            TimedeltaArray(arr.reshape(2, 2))
+
+        with pytest.raises(ValueError, match="Only 1-dimensional"):
+            # 0-dim
+            TimedeltaArray(arr[[0]].squeeze())
+
```

Comment:
Hmm...part of me wants to parameterize this...but I'll leave it to you because it's not the nicest of parameterizations compared to others.